src/react/hooks/use-presence.ts

```diff
@@ -75,6 +76,9 @@ export const usePresence = (params?: UsePresenceParams): UsePresenceResponse =>
   // store the roomStatus in a ref to ensure the correct value is used in the effect cleanup
   const roomStatusAndConnectionStatusRef = useRef({ roomStatus, connectionStatus });
 
+  // create a stable reference for the onDiscontinuity listener
+  const onDiscontinuityRef = useEventListenerRef(params?.onDiscontinuity);
```

Comment:
I can't see anywhere in the `typedocs` that specifies the new behaviour of the hooks. I can see it's in the readme, should we make it explicitly clear in each hook too?

Reply:
Yes, I've documented this on the `StatusParams` type.